dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

```diff
@@ -270,6 +270,7 @@ std::unordered_map<TableID, SelectQueryInfo> DAGStorageInterpreter::generateSele
             analyzer->getPreparedSets(),
             analyzer->getCurrentInputColumns(),
             context.getTimezoneInfo());
+        query_info.req_id = log->identifier();
```

Comment:
For `PartitionTableScan`, do we need to distinguish between different `physical_table_id`?
such as `MPP<query:432340111618473985,task:1> Table<${physical_table_id}>`?